libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -1236,7 +1236,8 @@ def add_version_header(tc):
         },
         {
             "name": "__cpp_lib_three_way_comparison",
-            "values": {"c++20": 201907},
+            "values": {"c++20": 201711},
+            # {"c++20": 201711} # P1614R2 The Mothership has Landed (see P1902R1 Missing feature-test macros 2017-2019)
```

Comment:
I don't understand this comment.

Reply:
Basically P1614R2 sets a different FTM instead of bumping this one. P1902R1 changes P1614R2 to bump this FTM instead.

Reply:
So should this say `201907` (which is the value in P1902R1)?

```suggestion
            # {"c++20": 201907} # P1614R2 The Mothership has Landed (see P1902R1 Missing feature-test macros 2017-2019)
```

Reply:
Thanks! This seems copy-paste programming ;-)